hpi-archetype/pom.xml

```diff
@@ -15,15 +14,32 @@
   <!-- $ --><version>1.0-SNAPSHOT</version><!-- /$ -->
   <packaging>hpi</packaging>
 
+  <properties>
+    <!-- Baseline Jenkins version you use to build and test the plugin. Users must have this version or newer to run. -->
+    <jenkins.version>1.625.3</jenkins.version>
+    <!-- Java Level to use. Java 7 required when using core >= 1.612 -->
+    <java.level>7</java.level>
+    <!-- Baseline Jenkins version you use to build and test the plugin. -->
+    <jenkins-test-harness.version>2.0</jenkins-test-harness.version>
```

Comment:
…though https://github.com/jenkinsci/jenkins-test-harness/pull/2 is pretty critical.